slf4j-api/src/test/java/org/slf4j/helpers/UtilTest.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2004-2011 QOS.ch
```

Comment:
This particular class has been deleted, but there are several new classes now and I've updated the copyright date there.